src/appmixer/intelswift/core/NewContact/NewContact.js

```
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = {
+
+    async start(context) {
+
+        const { data } = await makeRequest(context, '/api/v1/accounts/account', { method: 'POST' });
+        const tenantId = data?.data[0].tenantId;
+
+        context.log({ stage: 'profile info', info: context, tenantId, check: "1" });
```

ESLint

[error] Strings must use singlequote.

(quotes)